drivers/SmartThings/matter-appliance/profiles/extractor-hood-ac-light-binary.yml

```diff
@@ -0,0 +1,30 @@
+name: extractor-hood-ac-light-binary
+components:
+- id: main
+  label: Main
+  capabilities:
+  - id: fanMode
+    version: 1
+  - id: fanSpeedPercent
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: KitchenHood
+- id: activatedCarbonFilter
+  label: Activated carbon filter
+  capabilities:
+  - id: filterState
+    version: 1
+  - id: filterStatus
+    version: 1
+  categories:
+  - name: KitchenHood
+- id: switch
```

Comment:
I think `light` would be a more appropriate name for component

Reply:
Updated as suggested in latest commit.